src/pages/WalletPage.vue

```diff
@@ -339,6 +364,7 @@ export default {
       "setProofs",
       "getKeysForKeyset",
     ]),
+    ...mapActions(useReceiveStore, ["knowThisMintOfTokenJson"]),
```

Comment:
The following change fixes the following error:

```
ReferenceError: Can't find variable: useReceiveStore
```


```suggestion
    ...mapActions(useReceiveTokensStore, ["knowThisMintOfTokenJson"]),
```